### PR TITLE
Implement etherbi fetcher logic in a module to be resued

### PR DESCRIPTION
**Pull request by IvanIvanoff**
NOTE: When adding the new env variables rename `ETHERBI_TRANSACTIONS_ENABLED` to `ETHERBI_TRANSACTIONS_SYNC_ENABLED`.

This PR introduces a module `EtherbiGenServer` which injects behaviour that abstracts away the repeating code for starting and initializing the genserver, creating influxdb and repeating the work at given time intervals.

### CODE REVIEW

File: config/test.exs [hunk 1]
```diff
@@ -50,7 +50,7 @@ config :sanbase, Sanbase.Github.Store,
 config :sanbase, Sanbase.ExternalServices.TwitterData.Store,
   database: "twitter_followers_data_test"
 
-config :sanbase, Sanbase.Etherbi.Store,
+config :sanbase, Sanbase.Etherbi.Transactions.STore,
```
Comment (valo) [inline config/test.exs]: Typo: `STore` -> `Store`

Comment (IvanIvanoff) [reply config/test.exs]: Fixed but still have not pushed it

File: lib/sanbase/etherbi/burn_rate.ex [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+  alias Sanbase.Etherbi.BurnRate.Store
+  alias Sanbase.Repo
+  alias Sanbase.Model.Project
+
+  def work() do
+    # Precalculate the number by which we have to divide, that is pow(10, decimal_places)
+    token_decimals = build_token_decimals_map()
+
+    query = from(p in Project, where: not is_nil(p.ticker), select: p.ticker)
```
Comment (valo) [inline lib/sanbase/etherbi/burn_rate.ex]: I think we can narrow down the query here. We can try to extract this query https://github.com/santiment/sanbase2/blob/master/lib/sanbase_web/graphql/resolvers/project_resolver.ex#L55 into a common place and reuse that

File: lib/sanbase/etherbi/transaction_volume.ex [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+      tickers,
+      &fetch_and_store(&1, token_decimals),
+      max_concurency: 1,
+      timeout: 170_000
+    )
+    |> Stream.run()
+  end
+
+  def fetch_and_store(ticker, token_decimals) do
+    with {:ok, transaction_volumes} <- Fetcher.transaction_volume(ticker) do
```
Comment (valo) [inline lib/sanbase/etherbi/transaction_volume.ex]: I think we can skip fetching the `transaction_volume` if we don't have the `token_decimals` for the project, right? We won't store any data in that case anyway.

Comment (IvanIvanoff) [reply lib/sanbase/etherbi/transaction_volume.ex]: Or even better - add `not is_nil(p.token_decimals)` when fetching the tickers and avoid explicit checking here.

Comment (valo) [reply lib/sanbase/etherbi/transaction_volume.ex]: Yep. Let's do it like this

File: lib/sanbase/etherbi/burn_rate.ex [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+  # Better return no information than wrong information. If we have no data for the
+  # number of decimal places `nil` is written instead and it gets filtered by the Store.import()
+  defp convert_to_measurement(
+         burn_rates,
+         ticker,
+         token_decimals
+       ) do
+    burn_rates
+    |> Enum.map(fn {datetime, burn_rate} ->
+      if decimal_places = Map.get(token_decimals, ticker) do
```
Comment (valo) [inline lib/sanbase/etherbi/burn_rate.ex]: Do we need this check now?

Comment (IvanIvanoff) [reply lib/sanbase/etherbi/burn_rate.ex]: Not really, pushed a change.

**Pull request merged**